build.gradle

```diff
@@ -90,7 +90,7 @@ dependencies {
 
     api group: 'org.elasticsearch', name: 'elasticsearch', version: '7.17.14'
 
-    implementation group: 'com.github.hmcts', name: 'et-data-model', version: '4.0.0'
+    implementation group: 'com.github.hmcts', name: 'et-data-model', version: '3.0.7'
```

Comment:
why is this being downgraded?

Reply:
4.0.0 was implemented by someone not in our team and I based my branch from master. Normally we had 3.0.6 continuing. Master branch does not have 4.0.0 updates. As team we are always implementing using the main version branch as 3.x.x. 

et-common works fine with that branch and maybe it was implemented by @aaronjduffy and used in et-commons for testing reasons. 

![Screenshot 2023-11-06 at 10 04 00](https://github.com/hmcts/et-common/assets/105645217/7bad4b31-0c18-429d-9211-7ad94352243e)